src/App.test.tsx

```diff
@@ -1,9 +1,11 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import App from './App'
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+  render(<App />)
+  const linkElement = screen.getByText(/learn react/i)
+  // TODO: properly type linkElement so that toBeInDocument doesn't throw a type error
+  // @ts-expect-error
```

Comment:
eslint breaks here because of the `@typescript-eslint/no-unused-vars` rule, that says that this annotation is unused.

You could just remove this line or remove this entire file, since it is just boilerplate that doesn't actually test anything